### PR TITLE
Kildevisning design

### PR DESCRIPTION
**Pull request by hypesystem**
Visning af kilder. Er responsive.

Kendte mangler:
- felter skal tilpasses
- når vi har adgang til fancy font skal "Kilde"-teksten skrives med den, i stedet for at være et indsat billede.

image

image

image

### CODE REVIEW

File: src/app/person-appearance/person-appearance.component.html [hunk 1]
```diff
@@ -1,114 +1,152 @@
...
+    <div class="person-appearance-end-info">
+        <div class="person-appearance-end-info__entry">
+            <div class="person-appearance-end-info__entry__label">Kilde ID</div>
+            <a href="/pa/{{ this.pa.id }}">{{ this.pa.id }}</a>
+        </div>
+        <!--div class="person-appearance-end-info__entry">
+            <div class="person-appearance-end-info__entry__label">Senest opdateret</div>
+        </div-->
+    </div>
 </div>
```
Comment (Arth101) [inline src/app/person-appearance/person-appearance.component.html:152]: Missing newline.

File: src/assets/styling/rows-and-columns.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  @for $i from 2 through 6 {
+    &--gap-#{$i} {
+      grid-gap: #{$i}rem;
+    }
+  }
+
+  @each $name, $maxWidth in $breakpoints {
+    @media (min-width: $maxWidth) {
+      @for $i from 2 through 6 {
+        &--gap-#{$i}--#{$name} {
```
Comment (Arth101) [inline src/assets/styling/rows-and-columns.scss:16]: BEM does not allow multiple modifiers (ie classes should not be named with `--` multiple times.
Do we need to have specific gaps for specific display sizes?

File: src/app/person-appearance/person-appearance.component.html [hunk 3]
```diff
@@ -1,114 +1,152 @@
...
+    font-weight: 400;
+    padding-right: 1rem;
+    width: 33%;
+}
 
+.person-appearance-end-info__entry {
+    font-size: 1rem;
+    font-weight: 400;
+}
+.person-appearance-end-info__entry__label {
```
Comment (Arth101) [inline src/app/person-appearance/person-appearance.component.html:56]: A better naming could be something like:
```
.person-appearance__footer
.person-appearance__footer-entry
.person-appearance__footer-label
```

File: src/app/person-appearance/person-appearance.component.html [hunk 4]
```diff
@@ -1,114 +1,152 @@
...
-    </div>
-
-    <!-- <div class ="row">
-        <div class="col-sm">
-            <div>Alder: {{pa.age}}</div>
-            <div>Stilling: {{pa.Erhverv}}<div>
-            <div>Handicaps: -</div>
+    <div class="lls-card lls-card--census">
+        <div class="lls-row lls-row--gap-3--md lls-row--gap-5--lg lls-row--gap-6--xl">
+            <div class="lls-columns-12 lls-columns-6--md">
```
Comment (Arth101) [inline src/app/person-appearance/person-appearance.component.html:89]: Indentation is weird

File: src/assets/styling/rows-and-columns.scss (hunk 2 above)
Comment (hypesystem) [reply src/assets/styling/rows-and-columns.scss:16]: Yeahhh I think we need these. What should I do instead, then? `.lls-row--gap-2-md`, for example? I kinda like `--gap-2--md` 🐸

Comment (hypesystem) [reply src/assets/styling/rows-and-columns.scss:16]: Gonna merge for now, but maybe we can fix this later :smile:

**Pull request merged**